.travis.yml

```diff
@@ -1,16 +1,16 @@
-sudo: false
+sudo: true
```

Comment:
Maybe better. (Even `sudo: true` looks working)
Fixed in https://github.com/mackerelio/mackerel-agent/pull/330/commits/e018b7a40494dd7f10254f99f65ba5a9d342734d.

Build details of https://github.com/mackerelio/mackerel-agent/pull/330/commits/5c8d2e21dbe8f809df4a3ef16c974be98274dc60 shows this branch can build rpms with Docker. (Building rpms in `after-script` is re-reverted on https://github.com/mackerelio/mackerel-agent/pull/330/commits/e7055122568d0adca7be2331da9c515d5ac0d3e7)